micromamba/tests/test_info.py

```diff
@@ -78,3 +78,40 @@ def test_not_env(tmp_home, tmp_root_prefix, prefix_selection, existing_prefix):
     assert f"env location : {location}" in infos
     assert f"user config files : {tmp_home / '.mambarc'}" in infos
     assert f"base environment : {tmp_root_prefix}" in infos
+
+
+@pytest.mark.parametrize("base_flag", ["", "--base"])
+@pytest.mark.parametrize("prefix_selection", [None, "prefix", "name"])
+def test_base_subcommand(tmp_home, tmp_root_prefix, prefix_selection, base_flag):
+    os.environ["CONDA_PREFIX"] = str(tmp_root_prefix)
```

Comment:
`pytest` exposes a `monkeypatch` fixture which can be used to [monkeypatch environment variable in the scope of a test](https://docs.pytest.org/en/stable/how-to/monkeypatch.html#monkeypatching-environment-variables). This is a bit safer and the intend is clearer.

```suggestion
def test_base_subcommand(tmp_home, tmp_root_prefix, prefix_selection, base_flag, monkey_patch):
    monkeypatch.setenv("CONDA_PREFIX", str(tmp_root_prefix))
```

Not that this suggestion might not respect linters' formatting.